dts/bindings/gpio/stemma-qt-connector.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2024 TOKITA Hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    STEMMA QT is a 1.0mm-JST-SH connector for I2C devices.
+    The pin assignments are as follows.
+
+        0  SCL
+        1  SDA
+        -  VCC (3-5V)
+        -  GND
```

Comment:
why did you skip the pin numbers for 2 and 3?

Reply:
This follows the same notation as the grove.
https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/gpio/grove-header.yaml#L9-L12

Other connectors also use this notation.

Reply:
Oh it's the pin id in the nexus node, of course! Ok nevermind sorry for the noise.